coralproject/talk
# [CORL-1616/18/28] Fix typo-ed or missing translation keys in admin/stream.ftl

## What does this PR do?

Fix typo-ed or missing translation keys in admin/stream.ftl

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## How do I test this PR?

Ctrl + F for the key changes in this PR and confirm they match their use in the stream/admin codebase
